api.go

```diff
@@ -224,24 +236,40 @@ func (api *API) addResource(prototype jsonapi.MarshalIdentifier, source CRUD, ma
 	}
 
 	api.router.Handle("OPTIONS", api.prefix+name, func(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
+		c := api.contextPool.Get().(*APIContext)
```

Comment:
I really like the usage of a sync.Pool. Never done that before but I looked into the gin code and they use it too. It makes sense because otherwise a APIContext struct would have to be allocated for every request and then garbage collected. In that way, it can be used again without new allocation/garbage collection, right?

When everything is working in api2go we should look for more performance optimisation stuff :)